src/components/Kv/KvMap.vue

```diff
@@ -260,21 +307,64 @@ export default {
 				center: [this.lat, this.long],
 				zoom: this.initialZoom || this.zoomLevel,
 				// todo make props for the following options
-				dragging: false,
-				zoomControl: false,
+				dragging: this.allowDragging,
+				zoomControl: this.showZoomControl,
 				animate: true,
 				scrollWheelZoom: false,
 				doubleClickZoom: false,
 				attributionControl: false
 			});
 			/* eslint-disable quotes */
 			// Add our tileset to the mapInstance
-			L.tileLayer('https://api.maptiler.com/maps/bright/{z}/{x}/{y}.png?key=n1Mz5ziX3k6JfdjFe7mx', {
+			let tileLayer = 'https://api.maptiler.com/maps/landscape/{z}/{x}/{y}.png?key=n1Mz5ziX3k6JfdjFe7mx';
+			if (this.showLabels) {
+				tileLayer = 'https://api.maptiler.com/maps/bright/{z}/{x}/{y}.png?key=n1Mz5ziX3k6JfdjFe7mx';
+			}
+			L.tileLayer(tileLayer, {
 				tileSize: 512,
 				zoomOffset: -1,
 				minZoom: 1,
-				crossOrigin: true
+				crossOrigin: true,
 			}).addTo(this.mapInstance);
+
+			if (this.countriesData.length > 0) {
+				const countriesFeatures = countriesBorders.features ?? [];
+
+				countriesFeatures.forEach((country, index) => {
+					const countryData = this.countriesData.find(data => data.isoCode === country.properties.ISO_A2);
+					if (countryData) {
+						countriesFeatures[index].lenderLoans = countryData.value;
+						countriesFeatures[index].numLoansFundraising = countryData.numLoansFundraising;
+					}
+				});
+
+				L.geoJson(
+					countriesBorders,
+					{
+						style: this.countryStyle,
+						onEachFeature: this.onEachCountryFeature
+					}
+				).addTo(this.mapInstance);
+
+				this.countriesData.forEach(country => {
+					if (country.numLoansFundraising > 0 && this.showFundraisingLoans) {
+						const circle = L.circle([country.lat, country.long], {
+							color: kvTokensPrimitives.colors.black,
+							weight: 1,
+							fillColor: kvTokensPrimitives.colors.brand[900],
+							fillOpacity: 1,
+							radius: 130000,
```

Comment:
Why a large number for the `radius`?

Reply:
This radius is in meters based on the map scale:
https://leafletjs.com/reference.html#circle-radius